.github/workflows/ci.yml

```
@@ -81,7 +78,7 @@ jobs:
       matrix:
         include:
           - node-version: 16
-            os: ubuntu-latest
+            os: windows-2019
```

if we're going to use 2019 we might want to add a link to the issue
```suggestion
            os: windows-2019 # slowness reported on newer versions https://github.com/actions/runner-images/issues/5166
```